apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/staticelement/PageStaticController.java

```diff
@@ -94,4 +99,35 @@ public DeleteStaticResponse deleteStaticElement(
                 : new DeleteStaticResponse("delete fail");
     }
 
+    @PostMapping("/{id}/update/hyperlink")
+    public UpdateStaticResponse updateHyperlink(
+            @PathVariable("page") Long pageId,
+            @PathVariable("id") Long componentId,
+            @RequestBody UpdateStaticRequests.UpdateHyperlink request,
+            @ApiIgnore @AuthenticationPrincipal final NbsUserDetails details
+    ) {
+        return pageStaticUpdater.updateHyperlink(componentId, request, details.getId());
+    }
+
+    @PostMapping("/{id}/update/read-only-comments")
+    public UpdateStaticResponse updateReadOnlyComments(
+            @PathVariable("page") Long pageId,
+            @PathVariable("id") Long componentId,
+            @RequestBody UpdateStaticRequests.UpdateReadOnlyComments request,
+            @ApiIgnore @AuthenticationPrincipal final NbsUserDetails details
+    ) {
+        return pageStaticUpdater.updateReadOnlyComments(componentId, request, details.getId());
+    }
+
+    @PostMapping("/{id}/update/default")
```

Comment:
Question: What are your thoughts on just having multiple routes map to the same method for the `default`?

I think you can do 
```java
@PutMapping({"/{id}/admin-comments", "/{id}/read-only"})
```